docs/Makefile

```diff
@@ -21,26 +21,36 @@ endif
 
 all: deps api build
 
-api:
-	$(QUARTODOC) build
-	$(QUARTODOC) interlinks
+ensure-dev:
+	$(MAKE) -C .. dev
+
+api: ensure-dev
+	@echo "::group::quartodoc interlinks"
```

Comment:
Smart. Just curious... does the `::group::task` nomenclature come from somewhere?

Reply:
It's a GHA thing. Gives it the little drop down to consolidate stdout/stderr within a single step. There can be many output groups per step.

Reply:
Whoa! That's awesome. TIL